python/llm/src/bigdl/llm/llm_patching.py

```diff
@@ -20,6 +20,7 @@
 from bigdl.llm.utils.common import invalidInputError
 
 is_bigdl_patched = False
+is_train_patched = False  # Specifically to check finetuning patch
```

Comment:
I think you can combine these two variables into one

Reply:
> I think you can combine these two variables into one

If we need to add inference patch later, we still need different variables to handle. I think maybe we could keep it also. Currently, if `train=False`, nothing is done during patching.

Reply:
You can have one variable "bigdl_patched" as enum (None, Train, Inference).

Reply:
> You can have one variable "bigdl_patched" as enum (None, Train, Inference).

OK, I'll use this instead.